pkg/kapp/clusterapply/changes_view.go

```diff
@@ -93,16 +89,13 @@ func (v *ChangesView) Print(ui ui.UI) {
 		}
 
 		if resource.IsProvisioned() {
-			condVal := cmdcore.NewConditionsValue(resource.Status())
 
 			row = append(row,
-				// TODO erroneously colors empty value
-				uitable.ValueFmt{V: condVal, Error: condVal.NeedsAttention()},
+
```

Comment:
nit: Remove new line.
I think you added a new line in all the places while removing the comments.

Reply:
Removed